.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
I think I should not use npx here...

Reply:
npx is bundled but lint-staged is not. In other words - what is so different about lint-staged dev dependency that we should not put it into dev dependencies as others?
Or should we use all dev-dependencies if possible via npx?
And then if we already installed it locally - why ask npx to search it somewhere if we have it installed?

Reply:
I see, then maybe try to replace it with `npm run` and lint-staged script?